src/sass/app_2/_common/components/calendar.scss

```diff
@@ -145,18 +144,82 @@
         }
         &--today {
             font-weight: 400;
-            border: 1px solid $COLOR_LIGHT_BLACK_1;
-            border-radius: 5px;
+            background: $COLOR_DARK_BLUE_5;
+
+            &:hover {
+                background: $COLOR_DARK_BLUE_5 !important;
+            }
+            &:before {
+                position: absolute;
+                content: '';
+                width: 100%;
+                left: 50%;
+                z-index: -1;
+                height: 100%;
+                padding-right: 10px;
+                background: $COLOR_DARK_BLUE_5 !important;
+            }
         }
         &--active {
             color: $COLOR_WHITE;
-            background: $COLOR_LIGHT_GRAY_1;
+            background: $COLOR_DARK_BLUE_5 !important;
             font-weight: bold;
 
             &:hover {
                 background: $COLOR_LIGHT_GRAY_1;
             }
         }
+        &--between {
+            background: $COLOR_DARK_BLUE_5;
+            font-weight: bold;
+            color: $COLOR_WHITE !important;
+
+            &-hover {
+                &:before {
+                    position: absolute;
+                    content: '';
+                    width: 100%;
+                    z-index: -2;
+                    height: 100%;
+                    padding-left: 28px;
+                    background: $COLOR_DARK_BLUE_4;
+                }
+            }
+            &:hover {
+                background: $COLOR_DARK_BLUE_5 !important;
```

Comment:
i think it's better to not use `!important`, colors changes can be done without `!important` if the rule was assigned correctly